tests/e2e/labelupdates.go

```diff
@@ -467,12 +467,9 @@ var _ bool = ginkgo.Describe("[csi-block-vanilla] [csi-block-vanilla-parallelize
 		ginkgo.By(fmt.Sprintf("Deleting pvc %s in namespace %s", pvc.Name, pvc.Namespace))
 		err = client.CoreV1().PersistentVolumeClaims(namespace).Delete(ctx, pvc.Name, *metav1.NewDeleteOptions(0))
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		// Waiting for some time for PVC to be deleted correctly
-		ginkgo.By(fmt.Sprintf("Sleeping for %v seconds to allow PVC deletion", oneMinuteWaitTimeInSeconds))
-		time.Sleep(time.Duration(oneMinuteWaitTimeInSeconds) * time.Second)
-
-		_, err = e2eVSphere.getLabelsForCNSVolume(pv.Spec.CSI.VolumeHandle,
-			string(cnstypes.CnsKubernetesEntityTypePVC), pvc.Name, namespace)
+		
+		ginkgo.By(fmt.Sprintf("Waiting for PVC %s to be deleted", pvc.Name))
+		err = waitForPVCDeletion(pv.Spec.CSI.VolumeHandle, pvc.Name, namespace)
```

Comment:
you can use https://github.com/kubernetes/kubernetes/blob/v1.25.0/test/e2e/framework/pv/pv.go#L745, and wait for pv to be in released state.